sdk/monitor/monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 # Release History
 
-## 1.0.0-beta.11 (Unreleased)
```

Comment:
There is some automatic process that updates this changelog every time there is a release, so we don't need to add unreleased one yet "beta.12"